zcash_client_sqlite/src/wallet/transparent.rs

```diff
@@ -494,6 +517,11 @@ pub(crate) fn transaction_data_requests<P: consensus::Parameters>(
     conn: &rusqlite::Connection,
     params: &P,
 ) -> Result<Vec<TransactionDataRequest>, SqliteClientError> {
+    // `lightwalletd` will return an error for `GetTaddressTxids` requests having an end height
+    // greater than the current chain tip height, so we take the chain tip height into account
+    // here in order to make this pothole easier for clients of the API to avoid.
+    let chain_tip_height = super::chain_tip_height(conn)?;
```

Comment:
Incidentally, why does the doc for `wallet::chain_tip_height` say
```rust
/// Returns the minimum and maximum heights of blocks in the chain which may be scanned.
```
when it actually only returns (optionally) a single height?

Reply:
Filed #1498.